python/tvm/relay/transform/transform.py

```diff
@@ -1386,3 +1368,51 @@ def SplitArgs(max_function_args):
         The registered pass for constant folding.
     """
     return _ffi_api.SplitArgs(max_function_args)
+
+
+def OutlineCompilerFunctionsWithExistingGlobalSymbols(compiler_filter=""):
+    """A pass to outline all literal functions in direct call positions which have a "Compiler"
```

Comment:
nit: numpydoc style and pep 257 wants a one-liner summary at the beginning of such multiline docstrings